src/RahaFirestore.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * A module that exports an api for interacting with Raha's Firestore db.
+ *
+ * Example usage:
+ * async function getAppliedOperations() {
+ *     return await get(
+ *         operationsCollectionFilters.applied(true)(
+ *             operationsCollection()));
+ * }
+ */
+
+import * as firebase from 'firebase';
+import 'firebase/firestore';
+
+let db;
+
+/**
+ * Retrieve an instance of the Firestore db.
+ */
+function getDb() {
+    if (!db) {
+        db = firebase.initializeApp(require('./config/firebase.config.json')).firestore();
```

Comment:
Why not just `const db = firebase.initializeApp(require('./config/firebase.config.json')).firestore();` outside like in most firebase example code?